src/main/java/frc/robot/Robot.java

```diff
@@ -104,53 +113,101 @@ public static AHRS getGyroscope() {
         autonRouteChooser.addOption("move backwards", AutonRoutes.GO_BACKWARD_OUT_OF_STARTING_ZONE);
+        autonRouteChooser.addOption("just shoot", AutonRoutes.JUST_SHOOT);
+
+        autonRouteChooser.addOption("shoot and back up", AutonRoutes.SHOOT_AND_BACK_UP_FROM_CENTER);
+        autonRouteChooser.addOption("shoot and back up forever", AutonRoutes.SHOOT_AND_BACK_UP_SKETCHILY);
+
+        autonRouteChooser.addOption("backup turn backup", AutonRoutes.BACKUP_TURN_BACKUP);
         autonRouteChooser.addOption("explode hidden bomb", AutonRoutes.BOOM);
         autonRouteChooser.addOption("shoot backup intake forward shoot", AutonRoutes.SHOOT_BACKUP_INTAKE_FORWARD_SHOOT);
-        autonRouteChooser.addOption("backup turn backup", AutonRoutes.BACKUP_TURN_BACKUP);
-        autonRouteChooser.addOption("shoot and back up", AutonRoutes.SHOOT_AND_BACK_UP);
 
         autonRouteChooser.addOption("Test old rotation PID", AutonRoutes.TEST_ROTATION);
         autonRouteChooser.addOption("Test new rotation PID", AutonRoutes.TEST_ROTATION_WITH_PID_COMMAND);
         autonRouteChooser.addOption("Test SmartMotion movement", AutonRoutes.TEST_SMART_MOTION_MOVEMENT);
+        autonRouteChooser.addOption(
+            "Test SmartMotion backward movement",
+            AutonRoutes.TEST_SMART_MOTION_BACKWARD_MOVEMENT
+        );
         autonRouteChooser.addOption("Test Manual PID movement", AutonRoutes.TEST_PID_MOVEMENT);
+        autonRouteChooser.addOption("Test Manual Backward PID movement", AutonRoutes.TEST_PID_BACKWARD_MOVEMENT);
+
+        //This is an emergency button for switching controllers mid match
+        SmartDashboard.putData(
+            "SWITCH CONTROLLERS",
+            new Command() {
+                {
+                    setName("Execute");
+                }
+
+                @Override
+                public void initialize() {
+                    System.out.println("We have switched thy controllers");
+                    int driverPort = driverController.getPort();
+                    int coDriverPort = coDriverController.getPort();
+                    driverController = new XboxController(coDriverPort);
+                    coDriverController = new XboxController(driverPort);
+                }
+
+                @Override
+                public boolean isFinished() {
+                    return true;
+                }
+            }
+        );
+
+        SmartDashboard.putNumber("Auton Delay (sec)", 0.0);
+
+        SmartDashboard.putNumber("Time Remaining", Timer.getMatchTime());
 
         SmartDashboard.putData("Auton Routes", autonRouteChooser);
 
+        SmartDashboard.putBoolean("Is AutoShooting?", false);
+
         driveLeftChild.follow(driveLeftParent);
         driveRightChild.follow(driveRightParent);
 
-        driveLeftParent.setInverted(true);
-        driveRightParent.setInverted(false);
+        driveLeftParent.setInverted(false);
+        driveRightParent.setInverted(true);
```

Comment:
We realized that the xbox stick forward is actually -1, so we inversed the motors and removed all the * -1's in our code.